DESCRIPTION

```diff
@@ -113,6 +112,8 @@ Suggests:
     ragg,
     showtext,
     sass
+Remotes:
+    rstudio/fontawesome
```

Comment:
Since it's going to be on CRAN soon, I think we can remove this and just merge once it's there.